### DISCUSSION TITLE
事实上valueEncoder或valueDecoder里的泛型是不是只有CacheValueHolder这一种可能？

### DISCUSSION THREAD

**Discussion by darknesstm** [alibaba/jetcache#767] [General]
如题，那么为啥要定义为
`Function<Object, byte[]>`
`Function<byte[], Object>`

不应该是这样的么？
`Function<CacheValueHolder<?>, byte[]>`
`Function<byte[], CacheValueHolder<?>>`

**Comment by areyouok** [comment]
应该没有为什么吧，已经这样写了
